src/modals/Login.jsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import Modal from "@mui/material/Modal";
+import LoginForm from "../components/forms/LoginForm";
+
+const style = {
+    position: "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: 400,
+    bgcolor: "background.paper",
+    border: "2px solid #000",
+    boxShadow: 24,
+    p: 4,
+};
+
+export default function Login() {
+    const [open, setOpen] = React.useState(false);
```

Comment:
do you usually write useState like this or is it from MUI thing?

Reply:
MUI thing